flink-tests/src/test/java/org/apache/flink/test/manual/CheckForbiddenMethodsUsage.java

```diff
@@ -44,6 +45,7 @@
  * <p>Forbidden calls include: - Byte / String conversions that do not specify an explicit charset
  * because they produce different results in different locales
  */
+@Ignore("broken test")
 public class CheckForbiddenMethodsUsage {
```

Comment:
I think it would be helpful to include the JIRA IDs for the tickets for these broken tests. Otherwise, people might lack the context.